tests/test.py

```diff
@@ -29,7 +29,7 @@ def str_to_class(string):
     return getattr(sys.modules[__name__], string)
 
 if __name__ == '__main__':
-    suite = unittest.TestSuite()
+    SUITE = unittest.TestSuite()
```

Comment:
Even if i isolate the into two functions (prepare_testclasses and prepare_testsuite) the problem with the constants persists. I don`t care if we disable the pylint checks as suggested in the link you posted (solution 2) or if we simply write the variable name in upper case letters.

the problem persists for

* suite 
* runner
* result